cxotime/scripts/print_time_conversions.py

```diff
@@ -0,0 +1,34 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import sys
+
+from cxotime import CxoTime
+
+
+def main(date=None, file=sys.stdout):
+    """Interface to entry_point script ``cxotime`` to print time conversions.
+
+    :param date: str, float, optional
+        Date for time conversion if sys.argv[1] is not provided. Mostly for testing.
+    :param file: file-like, optional
+        File-like object to write output (default=sys.stdout). Mostly for testing.
+    """
+    if date is None:
+        if len(sys.argv) > 2:
```

Comment:
No biggie, but I naively tried "cxotime --help" and "cxotime -help" and they weren't helpful as that just throws an exception `ValueError: Input values did not match any of the formats where the format keyword is optional`... 



So I'm not sure if there is some value to supporting some kind of usage help.